packages/chart/src/useChartTheme.tsx

```diff
@@ -0,0 +1,15 @@
+import { useContext } from 'react';
+
+import {
+  ChartThemeContext,
+  ChartThemeContextValue,
+} from './ChartThemeProvider';
+
+/**
+ * Hook to get the current `ChartThemeContextValue`.
+ */
+export function useChartTheme(): ChartThemeContextValue | null {
```

Comment:
Should we throw if there's no theme provided?

Reply:
I did this originally, but there were cases where the hook gets called before themes have loaded. I'll have to look at it again to remember the specifics

Reply:
I revisited this, and it seems to work to throw in `useChartTheme`. Must have been something else going on when I originally tested it.